api/desecapi/models.py

```diff
@@ -426,7 +426,6 @@ class Meta:
     'DNAME',  # "do not combine with DNSSEC", https://doc.powerdns.com/authoritative/settings.html#dname-processing
     'IPSECKEY',  # broken in pdns, https://github.com/PowerDNS/pdns/issues/9055 TODO enable support
     'KEY',  # Application use restricted by RFC 3445, DNSSEC use replaced by DNSKEY and handled automatically
-    'OPENPGPKEY',  # large records not supported https://github.com/desec-io/desec-slave/issues/34 TODO enable support
```

Comment:
also needs to be added to `SUPPORTED_RR_SET_TYPES` in `api/desecapi/tests/base.py`?